homeassistant/components/olarm/strings.json

```
@@ -0,0 +1,39 @@
+{
+  "config": {
+    "step": {
+      "device": {
+        "title": "Select device",
+        "description": "Choose the Olarm device you would like to integrate. If your device is not listed, please ensure you have enabled public API access in your Olarm account for that device.",
+        "data": {
+          "select_device": "Device",
+          "load_zones_bypass": "Load entities for zone bypassing?"
```

Checkbox labels should not have a question mark:

```suggestion
          "load_zones_bypass": "Load entities for zone bypassing"
```